File: spoj py/P148PROA.py
from sys import exit
N = int(input())
while N!=0:
    temp = N
    count = 1
    while temp!=1:
        if temp%2==0:
            temp/=2
        else:
            temp=temp*3+1
        count += 1
    print count
    N = int(input())


exit(0)

File: P148PROA.cpp
#include<iostream>
using namespace std;

int main(void)
{
	int N;
	cin>>N;
	while(N!=0)
	{
		int temp = N, count = 1;
		while(temp!=1)
		{
			if(temp%2==0)
				temp/=2;
			else
				temp=temp*3+1;
			count++;
		}
		cout<<count<<endl;
		cin>>N;
	}
 return 0;
}

